src/site/content/en/blog/fugu-status/index.md

```diff
@@ -21,42 +16,16 @@ security, privacy, trust, and other core tenets of the web.
 You can see the full list of new and potential capabilities at
 the [Fugu API Tracker](https://goo.gle/fugu-api-tracker).
 
-## Capabilities available behind a flag {: #flag }
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/uIIvM9xocYkjmBfHSrJE.svg", alt="A fugu fish, the logo of Project Fugu.", width="150", height="150" %}
```

Comment:
Can we put the image inline with the text so it's not floating in the middle of the page?

Reply:
Done.